src/jsDelivr/index.js

```diff
@@ -0,0 +1,94 @@
+import got from 'got';
+import log from '../log.js';
+import config from '../config.js';
+import datadog from '../datadog.js';
+
+const hits = new Map();
+
+function formatHits(pkg) {
```

Comment:
would now rename this & make it inline, so it would become just the next suggestion, and removing this function